egg/core/callbacks.py

```diff
@@ -169,6 +186,30 @@ def get_checkpoint(self):
             optimizer_state_dict=self.trainer.optimizer.state_dict(),
         )
 
+    def get_chk_files(self):
```

Comment:
What do you think about using _checkpoint_ instead of _chk_? It somehow confuses me with _chunk_

Reply:
done in 6e029ca